conf/application.conf

```diff
@@ -65,7 +65,7 @@ microservice {
     upload-customs-documents-frontend {
       protocol = http
       host = localhost
-      port = 10110
+      port = 10100
```

Comment:
Did you want to include this change to use the real UDF locally?

Reply:
Ah, yeah, I did this for some local proving.

What might be good, is to have both configs in there, with a feature toggle. So we can switch between calling the existing service and the new one at will.

For now, I think this one should go back to 10110. So I'll update that :)